recipe/meta.yaml

```diff
@@ -8,16 +8,18 @@ source:
   path : ..
 
 build:
-  number: 1
+  number: 2
   noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - lucid = lucid.launcher:main
```

Comment:
ooops... some leftovers here from the lucid recipe

Reply:
D'oh - thanks for catching that 😄